lms/static/js/spec/student_profile/learner_profile_view_spec.js

```diff
@@ -123,12 +123,7 @@ define(['backbone', 'jquery', 'underscore', 'common/js/spec_helpers/ajax_helpers
 
             beforeEach(function () {
                 loadFixtures('js/fixtures/student_profile/student_profile.html');
-                TemplateHelpers.installTemplate('templates/fields/field_readonly');
-                TemplateHelpers.installTemplate('templates/fields/field_dropdown');
-                TemplateHelpers.installTemplate('templates/fields/field_textarea');
-                TemplateHelpers.installTemplate('templates/fields/field_image');
                 TemplateHelpers.installTemplate('templates/fields/message_banner');
```

Comment:
I remembered the problem. The message banner is used by some non-optimized code as well (donations) so I was nervous about changing it to use RequireJS Text. Thinking it through some more, I think it will be okay as it will dynamically load the template (without caching) until that code is converted to use Optimizer too.